frontend/src/app/projects/models/model-source/t4c/manage-t4c-model/manage-t4c-model.component.ts

```diff
@@ -191,33 +186,32 @@ export class ManageT4CModelComponent implements OnInit, OnDestroy {
     }
   }
 
-  ngOnDestroy(): void {
-    this.t4cModelService._t4cModel.next(undefined);
-  }
-
   unlinkT4CModel() {
     if (!window.confirm(`Do you really want to unlink this T4C model?`)) {
       return;
     }
```

Comment:
I don't like these windows, they look a bit outdated and don't nicely integrate into our UI. We should get rid of them. Can also be handled in another PR.

Reply:
Created #792 for this